TestFoundation/TestNSArray.swift

```diff
@@ -46,7 +46,7 @@ class TestNSArray : XCTestCase {
             ("test_mutableCopying", test_mutableCopying),
             ("test_writeToFile", test_writeToFile),
             ("test_initWithContentsOfFile", test_initWithContentsOfFile),
-            ("test_readWriteURL", test_readWriteURL)
+            ("test_readWriteURL", test_readWriteURL),
```

Comment:
Although in principle having a trailing comma makes for less invasive changes in the future, since you aren't actually adding anything in this line then it's not relevant for this commit. Can you revert this line change and then squash the commits?